R/droplet-ssh.R

```diff
@@ -108,8 +108,14 @@ droplet_ip <- function(x) {
     stop("No network interface registered for this droplet\n  Try refreshing like: droplet(d$id)",
       call. = FALSE)
   }
-
-  v4[[1]]$ip_address
+  ips = do.call("rbind", lapply(v4, as.data.frame))
```

Comment:
change to use `<-` for assignment throughout your PR for consistency with this pkg

Reply:
Used styler to fix